code/framework/logback/src/main/resources/logback/logback.xml

```diff
@@ -185,6 +185,11 @@
         <level value="TRACE" />
         <appender-ref ref="FILE" />
     </logger>
+
```

Comment:
It's in a comment.  The reason I have it there is so I can easily uncomment it later if I need to troubleshoot.  I could just remove it.  It there for future lazy self.